src/quacc/calculators/mrcc/io.py

```diff
@@ -0,0 +1,126 @@
+from __future__ import annotations
+
+from io import StringIO
+from pathlib import Path
+
+from ase.io import read
+from ase.units import Hartree
+from ase.utils import reader, writer
+
+# Made from NWChem interface
+
+
+@reader
+def read_geom_mrccinp(fd):
+    """Method to read geometry from an MRCC input file."""
+    lines = fd.readlines()
+    atomslength = 0
+    atomslenline = -1
+
+    # Find geometry region of input file.
+    for index, line in enumerate(lines):
+        if "xyz" in line:
+            startline = index + 3
+            atomslenline = index + 1
+        elif index == atomslenline:
+            atomslength = int(line)
+            break
+
+    # Format and send to read_xyz.
+    xyz_text = "%i\n" % (atomslength)
+    xyz_text += " geometry\n"
+    for line in lines[startline : startline + atomslength]:
+        xyz_text += line
+    atoms = read(StringIO(xyz_text), format="xyz")
+    atoms.set_cell((0.0, 0.0, 0.0))  # no unit cell defined
```

Comment:
I think the `Atoms` object may still have PBCs? Might want to disable that with `atoms.pbc = False` if I'm right.